nix/rust-package.nix

```diff
@@ -12,6 +12,7 @@
 , lib
 , libiconv
 , makeSetupHook
+, mold
```

Comment:
Never heard of it, looks fast on paper, let's see then.

Reply:
indeed we will